lib/api/apiUtils/object/lifecycle.js

```diff
@@ -105,7 +105,44 @@ function processNonCurrents(bucketName, listParams, list) {
     return data;
 }
 
+function processOrphans(bucketName, listParams, list) {
+    const data = {
+        Name: bucketName,
+        Prefix: listParams.prefix,
+        MaxKeys: listParams.maxKeys,
+        IsTruncated: !!list.IsTruncated,
+        Marker: listParams.marker,
+        BeforeDate: listParams.beforeDate,
+        NextMarker: list.NextMarker,
+        Contents: [],
+    };
+
+    list.Contents.forEach(item => {
+        const v = item.value;
+        const versionId = (v.IsNull || v.VersionId === undefined) ?
+            'null' : versionIdUtils.encode(v.VersionId);
+        data.Contents.push({
+            Key: item.key,
+            LastModified: v.LastModified,
+            Etag: v.ETag,
+            Size: v.Size,
+            Owner: {
+                ID: v.Owner.ID,
+                DisplayName: v.Owner.DisplayName
+            },
+            StorageClass: v.StorageClass,
+            VersionId: versionId,
+            IsLatest: true, // for compatibily
```

Comment:
```suggestion
            IsLatest: true, // for compatibility
```
Also, maybe precise what kind of compatibility we're talking about here